devstack/init.sls

```diff
@@ -58,23 +58,25 @@ openstack-devstack config:
     - makedirs: True
     - mode: {{ devstack.mode }}
     - template: jinja
+    - env:
+      - STACK_PASSWORD: {{ devstack.password }}
+      - GIT_BASE: {{ devstack.conf.git_base }}
     - context:
         git_base: {{ devstack.conf.git_base }}
         branch: {{ devstack.repo.branch }}
         host_ip: {{ devstack.conf.host_ip }}
+        host_name: {{ devstack.conf.host_name or grains.host }}
         service_host: {{ devstack.conf.service_host }}
-        service_password: {{ devstack.conf.service_password }}
-        admin_password: {{ devstack.conf.admin_password }}
-        service_token: {{ devstack.conf.service_token }}
-        database_password: {{ devstack.conf.database_password }}
-        rabbit_password: {{ devstack.conf.rabbit_password }}
         enable_httpd_mod_wsgi_services: {{ devstack.conf.enable_httpd_mod_wsgi_services }}
         keystone_use_mod_wsgi: {{ devstack.conf.keystone_use_mod_wsgi }}
         logfile: {{ devstack.dir.log }}/{{ devstack.conf.logfile }}
+        logdays: {{ devstack.conf.logdays }}
         verbose: {{ devstack.conf.verbose }}
         enable_debug_log_level: {{ devstack.conf.enable_debug_log_level }}
         enable_verbose_log_level: {{ devstack.conf.enable_verbose_log_level }}
         reclone: {{ devstack.conf.reclone }}
+        odl_mgr_ip: {{ devstack.conf.odl_mgr_ip }}
```

Comment:
@noelmcloughlin, any particular reason to pass all the parameters individually, and not a single line like

```yaml
  - context:
      conf: {{ devstack.conf }}
```
and then, in [the jinja template](https://github.com/saltstack-formulas/devstack-formula/blob/master/devstack/files/localrc.j2) use `{{ conf.parameter }}`?

Reply:
Computers are hard!!

Reply:
:smile: